server/svix-server/src/core/types.rs

```diff
@@ -87,15 +87,15 @@ macro_rules! json_wrapper {
     ($name_id:ty) => {
         impl From<$name_id> for sea_orm::Value {
             fn from(v: $name_id) -> Self {
-                let v = serde_json::to_value(v).unwrap();
+                let v = serde_json::to_value(v).expect("Error serializing JSON");
                 Self::Json(Some(Box::new(v)))
             }
         }
 
         impl TryGetable for $name_id {
             fn try_get(res: &QueryResult, pre: &str, col: &str) -> Result<Self, TryGetError> {
                 match Json::try_get(res, pre, col) {
-                    Ok(v) => Ok(serde_json::from_value(v).unwrap()),
+                    Ok(v) => Ok(serde_json::from_value(v).expect("Error serializing JSON")),
```

Comment:
The error message both here and in the one above says "serializing" though one is `to_` and one is `from_`.